asset/cloud-client/quickstart_exportassets.py

```diff
@@ -31,7 +31,7 @@ def export_assets(project_id, dump_file_path):
     output_config = asset_service_pb2.OutputConfig()
     output_config.gcs_destination.uri = dump_file_path
     response = client.export_assets(parent, output_config)
-    print(response.result)
+    print(response.result())
```

Comment:
On line 26-27, please replace the double-quotes with single-quotes. LINT is not picking it up because the lines are commented out.

Reply:
On line 45, please avoid using the hanging indent. I recommend as:

```
parser.add_argument(
        'dump_file_path',
        help='The file ExportAssets will dump assets to, '
        'e.g.: gs://<bucket-name>/asset_dump_file')
```